### PR TITLE
DHFPROD-5830: fix for revert entity type

### PR DESCRIPTION
**Pull request by brucean52**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Review by timur-isangulov — APPROVED** [review]
Works fine, tested using scenario in the ticket.

**Review by Sanjeevani19 — APPROVED** [review]
One e2e test fails but not related to this PR so approving.

**Review by akshaysonvane — CHANGES_REQUESTED** [review]
I'm seeing some weird behavior on revert https://drive.google.com/file/d/1hA35mwK0KC-Jo2tIWE6j1NXNeZ04P15U/view

**Comment by brucean52** [comment]
> I'm seeing some weird behavior on revert https://drive.google.com/file/d/1hA35mwK0KC-Jo2tIWE6j1NXNeZ04P15U/view

Fixed this. Please review again

**Review by akshaysonvane — CHANGES_REQUESTED** [review]
Revert seems fine now. Seeing this issue while saving an entity: https://drive.google.com/file/d/1jsHFjVvQJtzT4xWwpzHrtfgFq6dv8JIo/view

**Comment by brucean52** [comment]
> Revert seems fine now. Seeing this issue while saving an entity: https://drive.google.com/file/d/1jsHFjVvQJtzT4xWwpzHrtfgFq6dv8JIo/view

Added the fix for saving an entity.

**Review by Sanjeevani19 — CHANGES_REQUESTED** [review]
The sort checkmark is still enabled even after saving those changes to database, Once page is refreshed the checkmark disappears.

**Comment by brucean52** [comment]
> The sort checkmark is still enabled even after saving those changes to database, Once page is refreshed the checkmark disappears.

Added the fix for Save All and updated e2e tests to cover the regression issues found.

**Review by Sanjeevani19 — DISMISSED** [review]

**Review by timur-isangulov — DISMISSED** [review]

**Review by akshaysonvane — CHANGES_REQUESTED** [review]
Revert and save looks good. Seeing this issue while updating the description: https://drive.google.com/file/d/1XHvNNqx7d5dmd5duLpyZgbHgBjTcVbgx/view

**Comment by brucean52** [comment]
> Revert and save looks good. Seeing this issue while updating the description: https://drive.google.com/file/d/1XHvNNqx7d5dmd5duLpyZgbHgBjTcVbgx/view

Added fix for entity description editing and added e2e test to cover this case.

**Review by Sanjeevani19 — APPROVED** [review]

File: marklogic-data-hub-central/ui/src/components/modeling/entity-type-table/entity-type-table.tsx [hunk 1]
```diff
@@ -215,7 +222,7 @@ const EntityTypeTable: React.FC<Props> = (props) => {
         let parseText = text.split(',');
         let entityName = parseText[0];
         let entityDescription = parseText[1];
-
+        console.log('entity desc', entityDescription)
```
Comment (akshaysonvane) [inline marklogic-data-hub-central/ui/src/components/modeling/entity-type-table/entity-type-table.tsx:225]: Can we get rid of this?

**Review by Sanjeevani19 — APPROVED** [review]

**Review by akshaysonvane — APPROVED** [review]
This looks really good.

**Pull request merged**